src/test/java/roomescape/MissionStepTest.java

```diff
@@ -49,4 +48,76 @@ void setUp() {
 
         assertThat(token).isNotBlank();
     }
+
+    @Test
+    void 이단계() {
+
+        String token = createToken("admin@email.com", "password");
+
+        Map<String, String> params = new HashMap<>();
+        params.put("date", "2024-03-01");
+        params.put("time", "1");
+        params.put("theme", "1");
```

Comment:
이건 개인 취향의 영역이긴 한데, `Map` 객체는 다음과 같이도 생성할 수 있어요.
```suggestion
        Map<String, String> params = Map.of("date", "2024-03-01",
                "time", "1",
                "theme", "1");
```

Reply:
이렇게 작성하는게 더 깔끔해 보이네요. 감사합니다 😄